docs/plugins.md

```diff
@@ -31,7 +31,16 @@ When plugins cannot be found automatically, you can load them with:
   });
   ```
 
-Prettier expects each of `pluginSearchDirs` to contain `node_modules` subdirectory, where `@prettier/plugin-*`, `@*/prettier-plugin-*` and `prettier-plugin-*` will be searched. For instance, this can be your project directory or the location of global npm modules.
+- Or the [Configuration File](configuration.md):
+
+  ```json
+  {
+    "pluginSearchDirs": ["./dir-with-plugins"],
+    "plugins": ["prettier-plugin-foo"]
+  }
+  ```
+
+The path that is provided to `pluginSearchDirs` will be searched for `@prettier/plugin-*`, `prettier-plugin-*`, and `@*/prettier-plugin-*`. For instance, this can be your project directory, a `node_modules` directory, the location of global npm modules, or any arbitrary directory that contains plugins.
```

Comment:
As we are here, can you add something to clarify that items in `plugins` won't be searched in `pluginSearchDirs` or anywhere, we simply `require()` it, it can be a package name or path to the plugin(whatever `require()` takes), I'm not a native English speaker, feel free to write.

Reply:
Hopefully the explanation I added is clear. Let me know if you think it's not clear/doesn't make sense.